internal/host/hostcommands/inventory_cmd.go

```diff
@@ -26,6 +29,9 @@ func (h *inventoryCmd) GetSteps(ctx context.Context, host *models.Host) ([]*mode
 			host.ID.String(),
 		},
 	}
+	if h.enableVirtualInterfaces {
```

Comment:
why do we need to send it to the agent? 
I was thinking that the agent will send everything and the service will decide if it's need to be used or not.

Reply:
For backwards compatibility - if the agent is newer than the service and sends all the interfaces, the service will use the first interface (hard-coded) which is not always a useable one. 

Therefore we needed a way for the new agent to tell when the service is new enough to handle all the interfaces being sent.

Slack discussion for context: https://coreos.slack.com/archives/C014N2VLTQE/p1654481131218899

Reply:
i suspect that this is not backward compatible, we can have a case of new service old agent not the other way around